src/sentry/sentry_metrics/consumers/indexer/processing.py

```diff
@@ -63,7 +64,13 @@ def process_messages(
         The value of the message is what we need to parse and then translate
         using the indexer.
         """
-        batch = IndexerBatch(self._config.use_case_id, outer_message)
+        should_index_tag_values = (
+            options.get(self._config.index_tag_values_option_name)
+            if self._config.index_tag_values_option_name
+            else True
+        )
+
+        batch = IndexerBatch(self._config.use_case_id, outer_message, should_index_tag_values)
```

Comment:
Since we either index all orgs or None, evaluation of the option happens before building the `IndexerBatch`. If no option name is set we set the default to `True` for cases like `release-health`